uwsgi.h

```diff
@@ -1356,8 +1356,11 @@ struct wsgi_request {
 	char *appid;
 	uint16_t appid_len;
 
-	//this is big enough to contain sockaddr_in
-	struct sockaddr_un c_addr;
+	union address {
```

Comment:
Changes at the end of struct please for ABI compatibility, so in this case move the union at the end and switch the c_addr callers to the new field i suppose.